src/opower/utilities/bwp.py

```diff
@@ -0,0 +1,48 @@
+"""Burbank Water and Power (BWP)."""
+
+from typing import Optional
+
+import aiohttp
+
+from ..const import USER_AGENT
+from .base import UtilityBase
+
+# from .helpers import async_auth_saml
+
+
+class BWP(UtilityBase):

```

Comment:
The utility website is burbankwaterandpower.com so rename the file to burbankwaterandpower.py and the class to BurbankWaterAndPower.